README.md

```diff
@@ -24,9 +24,19 @@ https://ristretto.group/
 ## Status
 
 This library is in an extremely early stage of development and is not ready to
-be used yet.
+be used yet. Check back later for updates.
 
-Check back later for updates.
+### Known Issues
+
+#### Travis CI
+
+* Sporadic SEGV on Travis C with clang when running `ristretto_gen_tables`, e.g.
```

Comment:
😭

Reply:
seems ... less than ideal

Reply:
It gets worse:

https://github.com/Ristretto/libristretto255/pull/17/files#diff-be5c31b26c462f22ad555004082e6eaaR433